.travis.yml

```diff
@@ -97,7 +97,7 @@ jobs:
           npm ci
           npm install kuzzle-sdk-v7@$TRAVIS_BUILD_DIR
           cd test/e2e/run-test && npm ci && cd -
-          CYPRESS_RETRIES=5 npm run test:e2e
```

Comment:
End 2 end tests must be run with the `test:e2e` command. You should change the `package.json` if it's not the case

Reply:
That means a PR on admin console repo and merge to master https://github.com/kuzzleio/kuzzle-admin-console/pull/829